packages/core/src/etherscan.ts

```diff
@@ -194,9 +178,16 @@ export const attemptVerification = async (
   minimumCompilerInput: SolcInput,
   provider: ethers.Provider,
   chainId: string,
-  etherscanApiKey: string
+  etherscanApiKey: string,
+  explorer?: ExplorerName
 ): Promise<{ success: boolean; message?: string }> => {
-  const { urls } = getChainConfig(Number(chainId))
+  const urls = fetchEtherscanConfigForNetwork(BigInt(chainId), explorer)
```

Comment:
Does this line throw an error if the network isn't supported by the DevOps platform? If so, will the Deploy command fail during verification for unsupported networks? We don't need to resolve that in this PR, but could you make a ticket for it?

Reply:
This line throws an error if verification (via either Blockscout or Etherscan) isn't supported on this network period, which I believe is the current behavior. 

It would definitely be better to error at the beginning of the deployment if the user requests verification on a network where we don't support verification. [I made a ticket for that.](https://linear.app/chugsplash/issue/CHU-662/error-before-the-deployment-starts-if-the-user-request-verification-on)